scripts/events_module/event_filters.py

```diff
@@ -0,0 +1,412 @@
+import re
+
+import ujson
+
+from scripts.game_structure.game_essentials import game
+from scripts.utility import (
+    get_alive_status_cats,
+    filter_relationship_type,
+)
+
+
+def event_for_location(locations: list) -> bool:
+    """
+    checks if the clan is within the given locations
+    """
+    if "any" in locations:
+        return True
+
+    for place in locations:
+        if ":" in place:
+            info = place.split(":")
+            req_biome = info[0]
+            req_camps = info[1].split("_")
+        else:
+            req_biome = place
+            req_camps = ["any"]
+
+        if req_biome == game.clan.biome.lower():
+            if "any" in req_camps or game.clan.camp_bg in req_camps:
+                return True
+            else:
+                return False
+            
+        return False
```

Comment:
```suggestion
        return False
```
Can just shorten it to this, since it doesn't matter where it fails the check, just that it fails it.